axelor-account/src/main/java/com/axelor/apps/account/service/fixedasset/FixedAssetLineMoveServiceImpl.java

```diff
@@ -473,14 +473,14 @@ public void generateDisposalMove(
       int transferredReason,
       LocalDate disposalDate)
       throws AxelorException {
-
     Journal journal = fixedAsset.getJournal();
     Company company = fixedAsset.getCompany();
     Partner partner = fixedAsset.getPartner();
     String origin =
         fixedAsset.getFixedAssetSeq() != null
             ? fixedAsset.getFixedAssetSeq()
             : fixedAsset.getReference();
+    Integer counter = 0;
```

Comment:
Please change the type to an `int` and change the variable name to "moveLineSequenceCounter"